Sources/SecureXPC/XPCServerEndpoint.swift

```diff
@@ -1,12 +1,18 @@
 //
 //  XPCServerEndpoint.swift
-//  
+//  SecureXPC
 //
 //  Created by Alexander Momchilov on 2021-11-28.
 //
 
 import Foundation
 
+/// An endpoint is used to create clients which can communicate with their associated server.
+///
+/// Endpoints are retrieved from a server's ``NonBlockingServer/endpoint`` property. They can be used in the same process or sent across an existing XPC
+/// connection.
+///
+/// > Warning: While ``XPCServerEndpoint`` conforms to `Codable` it can only be encoded and decoded by the `SecureXPC` framework.
```

Comment:
The Swift doc format has a style exactly for this:

```suggestion
/// - Warning:
```

Perhaps `Important` might be more suitable, IDK. You can see them all here: https://nshipster.com/swift-documentation/#additional-fields

Reply:
This is the [DocC format](https://developer.apple.com/documentation/xcode/formatting-your-documentation-content):
<img width="959" alt="image" src="https://user-images.githubusercontent.com/5177359/150466841-ef6404da-a275-404b-a86a-2a0d518382b2.png">

Reply:
TIL. Nice!